packages/manager/apps/cloud/client/app/dbaas/logs/detail/roles/logs-roles.service.js

```diff
@@ -163,6 +205,31 @@ class LogsRolesService {
       );
   }
 
+  addKibana(serviceName, roleId, kibana, permissionType) {
+    return this.PermissionsApiService.addKibana(
+      {
+        serviceName,
+        roleId,
+      },
+      {
+        kibanaId: kibana.kibanaId,
+        permissionType,
+      },
+    )
+      .$promise.then((operation) => {
+        this.RolesAapiService.resetAllCache();
+        return this.LogsHelperService.handleOperation(
+          serviceName,
+          operation.data || operation,
+        );
+      })
+      .catch((err) =>
+        this.LogsHelperService.handleError('logs_roles_add_kibana_error', err, {
+          tokenName: kibana.name,
+        }),
```

Comment:
throw err ?

Reply:
All methods from the following service are using a **HelperService** where notifications are displayed to the customer.

Throwing errors could be implemented in a separate Pull Request even I think.